lib/beaker/logger.rb

```diff
@@ -83,6 +83,35 @@ def initialize(*args)
 
       @destinations = []
 
+      @log_colors =  {
+        :error => RED,
+        :warn => BRIGHT_RED,
+        :success => MAGENTA,
+        :notify => BLUE,
+        :info => GREEN,
+        :debug => WHITE,
+        :trace => BRIGHT_YELLOW,
+        :perf => BRIGHT_MAGENTA,
+        :host => YELLOW
+      }
+
+      @log_colors.merge!(options[:log_colors]) if options[:log_colors]
+
+      # if a user overrides any of the log_colors, we will no longer
+      # override the colors at all on a CI build. This is b/c it is
+      # assumed that if a user is overriding the colors, they know
+      # what they are doing. We could potentially add an additional
+      # option a user can pass to be explicit about still allowing
+      # the override.
+      unless options[:log_colors]
+        # Jenkins exposed variable - should be present on the slave directing
+        # the beaker run
+        ci_build = ENV['BUILD_NUMBER'] != nil
+
+        @log_colors[:notify] = NORMAL if ci_build
+        @log_colors[:info] = NORMAL if ci_build
+      end
```

Comment:
Probably, but you mean lines 109-112. The comments still apply and should be left in there.